pkg/labels/labels.go

```diff
@@ -132,11 +135,44 @@ func (ls Labels) MatchLabels(on bool, names ...string) Labels {
 	return matchedLabels
 }
 
+// noescape hides a pointer from escape analysis.  noescape is
+// the identity function but escape analysis doesn't think the
+// output depends on the input. noescape is inlined and currently
+// compiles down to zero instructions.
+// USE CAREFULLY!
+// This was copied from the runtime; see issues 23382 and 7921.
+//go:nosplit
+//go:nocheckptr
+func noescape(p unsafe.Pointer) unsafe.Pointer {
```

Comment:
`noEscape` maybe? I'm aware it's a copied function still bothers me 😬 
```suggestion
func noEscape(p unsafe.Pointer) unsafe.Pointer {
```